src/views/List/List.jsx

```diff
@@ -17,11 +22,29 @@ export default function List({ token }) {
   }, []);
 
   return (
-    <ul>
-      {data.map((listItem, index) => {
-        const { name } = listItem;
-        return <li key={index}>{name}</li>;
-      })}
-    </ul>
+    <section>
+      <div className="div">
+        {data.length ? (
+          <ul>
+            {data.map((listItem, index) => {
+              const { name } = listItem;
+              return <li key={index}>{name}</li>;
+            })}
+          </ul>
+        ) : (
+          <>
+            <h1>Smart Shopping List</h1>
+            <div className="icons">
+              <SiProbot className="icon" />
```

Comment:
Again love how comfortable you are with using the right tools here with the UI library and making those decisions to add new things to the app, I haven't personally used React Icons before 🤖 really enjoying seeing your thoughts and the tools you are using